ui/app/AppLayouts/Chat/components/GroupInfoPopup.qml

```diff
@@ -223,10 +220,16 @@ ModalPopup {
                               contactRow.nickname : Utils.removeStatusEns(model.userName)
                     anchors.left: identicon.right
                     anchors.leftMargin: Style.current.smallPadding
-                    anchors.right: adminLabel.left
-                    anchors.rightMargin: Style.current.smallPadding
                     anchors.verticalCenter: parent.verticalCenter
                     font.pixelSize: 13
+                    StyledText {
+                        visible: model.userName === (profileModel.profile.preferredUsername || profileModel.profile.username)
```

Comment:
It would be more reliable to compare the pubkey
`model.pubKey === profileModel.profile.pubKey`